documentation/assemblies/assembly-managing-users-kube.adoc

```diff
@@ -1,24 +1,24 @@
 // Module included in the following assemblies:
 //
-//
+//master.adoc
 :parent-context: {context}
-:context: managing-users
-
-[id='managing-users-{context}']
+:context: managing-users-kube
+:cmdcli: kubectl
 
-= Managing users
+[id='managing-users-kube-{context}']
+= Managing users on Kubernetes
 
-{ProductName} user management is only supported when using the standard authentication service. Users can
+{ProductName} user management is only supported when using the `standard` authentication service. On Kubernetes, users can
 be managed using a Kubernetes native REST API and custom resources using the command-line tools.
 
 .Prerequisites
 
 * {ProductName} is installed with custom resources enabled
```

Comment:
I think we can remove this prerequisite, as it is enabled by default and unlikely to not be enabled.

Reply:
Done